python/lsst/ts/mthexapod/hexapod_csc.py

```diff
@@ -246,7 +246,11 @@ async def config_callback(self, client: hexrotcomm.CommandTelemetryClient) -> No
         client : `lsst.ts.hexrotcomm.CommandTelemetryClient`
             TCP/IP client.
         """
-        await self.evt_configuration.set_write(
+
+        # This is to keep the backward compatibility of ts_xml v22.0.0 that
```

Comment:
`backward compatibility of` -> `backward compatibility with`

Reply:
Fixed in the commit: [1d80bf8](https://github.com/lsst-ts/ts_mthexapod/pull/81/commits/1d80bf8d9e5add0ba14ef74ccfd3cce7efd80e76). Thanks!